docs/development_suite/api-console/api-design/merge_collaboration.md

```diff
@@ -5,12 +5,17 @@ sidebar_label: Merge of configurations
 ---
 
 :::caution
-Currently, in the merge process, the following Design sections are not supported:
+This feature is not a `git merge`, which means it does not include all the content of the project repository during the comparison between the two branches.
+
+Currently, it includes only the configurations included in the _Design_ sections of the Console with the exception of:
 
 - Visualize
 - Advanced
 
-Therefore, the user must manage them manually.
+This means that the Merge process will not allow you to compare the content coming from these two sections, as well as any other configuration included in the project repository but not in Console (such as pipelines for CI/CD operations or the content of the `mlp.yaml` file used by **mlp**).
```

Comment:
advanced are actually mergeable when using the Enhanced Workflow

Reply:
Nice to know. Gonna add this asap.

Reply:
Rephrased the content, the Advanced section indication now includes a specification whether the project have the Enhanced Workflow enabled or not

Reply:
great, thanks!